frontend/src/features/change-password/ChangePasswordModal.module.scss

```
@@ -71,6 +71,17 @@ white-space: nowrap;
 width: 408px;
 }
 
+.error {
+margin: -16px 0 20px;
+
+font-family: 'OpenSans', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)